HackGT/registration
# Add new Help Scout integration and Beacon functionality (#304)

* Added very basic initial functionality

* Implemented hmac hash checking (maybe?)

* Basic hello world to test signature validation in test deploy

* Add Help Scout secret

* Add more debugging

* Updated to latest version of typescript, fixed Help Scout signature validation

* Pushing some more changes to see how they look

* Make time zone questions required

* Testing some more

* Make timezone questions required

* Fix punctuation in dates

* Lots of Help Scout integration UI element improvements

* Finishing touches for Help Scout integration

* Help Scout Beacon support improvements

* Add info about Help Scout integration/Beacon to README

* Apparently you can't define secrets in both deployment.yaml and a Beehive yaml

* Remove package script that doesn't work

* Upgraded packages, including Handlebars

* Re-implemented Help Scout templates using Handlebars

* Show selected application/confirmation questions in Help Scout

* Cleanup Help Scout API security

* Remove extra lines

* A few final fixes and tweaks

* Make time zone questions required

* test file questions help scout view

* Fix file helpscout download URL

* actually fix help scout file links

* Remove some console.logs

* minor fixes

Co-authored-by: Evan Strat <evan10s@users.noreply.github.com>
Co-authored-by: Stephanie A <almeidasmail@gmail.com>
